_posts/2020-06-05-AlexisGroupB.md

```diff
@@ -0,0 +1,11 @@
+---
+title: the prompt of your choice e.g. "This summer I'm looking forward to..."
+author: Alexis Kirkman
+date: 2020-06-05
+---
+
+kayaking
+
+![kayaking](https://upload.wikimedia.org/wikipedia/commons/b/b2/Sea_Kayak.JPG)
+
+Comment your reply here.
```

Comment:
```suggestion
"Content length exceeded" when i click on your image, must be a super long kayak ride...
```

Reply:
@aakirkman10 commit tom's comment to your post